INSTALL.md

```diff
@@ -83,7 +84,7 @@ must be provided.
    `accumulo-site.xml`.  If your namenode is running at 192.168.1.9:9000
    and you want to store data in `/accumulo` in HDFS, then set
   `instance.volumes` to `hdfs://192.168.1.9:9000/accumulo`.
- * **Location of Zoookeeper and Hadoop jars** :  Setting `ZOOKEEPER_HOME` and
+ * **Location of Zookeeper and Hadoop jars** :  Setting `ZOOKEEPER_HOME` and
```

Comment:
I believe the correct capitalization is `ZooKeeper`. Someday, we should probably go through and make all our docs and method names consistent with this. I even saw one place the other day where we called it `zoo keeper` with a space!